op-devstack/sysgo/l2_cl.go

```diff
@@ -204,7 +204,8 @@ func WithL2CLNode(l2CLID stack.L2CLNodeID, isSequencer bool, l1CLID stack.L1CLNo
 				BeaconAddr: l1CL.beacon.BeaconAddr(),
 			},
 			Driver: driver.Config{
-				SequencerEnabled: isSequencer,
+				SequencerEnabled:   isSequencer,
+				SequencerConfDepth: 2,
```

Comment:
It would be beneficial to check L2 verifier also reorgs, by setting appropriate value for driver config's `VerifierConfDepth`.